site/content/blog/2021-03-29-zap-2-10-0-features/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "ZAP 2.10 Features"
+description: "A summary of the major features in ZAP release 2.10.0"
+summary: "Do you know what interesting bits were added to ZAP 2.10.0? Don't read release notes? This blog post is for you! 
+Dark mode, Expand/Collapse top panes, Custom pages, Scriptable encode/decode/hash, Authentication polling, Auth header via ENV vars, 
+Site tree control, and more."
+images:
+- https://www.zaproxy.org/blog/2021-01-31-zap-2-10-0-features/images/zapbot.png
+type: post
+tags:
+- blog
+- youdontknowzap
+date: "2021-03-29"
+addSocialPreview: true
+authors:
+- thorin
+
+---
+
+# ZAP 2.10.0
+
```

Comment:
Mention that 2.10.0 was released last year, link to the https://www.zaproxy.org/blog/2020-12-21-zap-2-10-0-the-10-year-anniversary-release/ blog but then say that this blog goes into more details?